tests/test_numerical_eda.py

```diff
@@ -0,0 +1,488 @@
+import altair as alt
+import numpy as np
+import pandas as pd
+from vega_datasets import data
+import warnings
+import pytest
+from simpler_eda.numerical_eda import numerical_eda
+
+cars = data.cars()
+
+
+@pytest.fixture
+def test_negative():
+    negative_data = pd.DataFrame(
+        {"x": [1, 2, 6, 7, 83, -1], "y": [1, 25, 6, -77, 2, 3]}
+    )
+    return negative_data
+
+
+def test_numerical_eda():
+
+    """
+    Tests the numerical eda function numerical_eda to make sure the outputs are
+    correctly rendering.
+
+    Returns:
+    --------
+    None
+        The test should pass and no asserts should be displayed.
+    """
+
+    # Unit test for a scatter plot with no transformations and using mark color
+    plot1 = numerical_eda(
+        cars,
+        xval="Horsepower",
+        yval="Acceleration",
+        color="red",
+        title="Scatter",
+        plot_type="scatter",
+        font_size=10,
+        color_scheme="purpleorange",
+        plot_height=100,
+        plot_width=200,
+    )
+    plot1_dict = plot1.to_dict()
```

Comment:
What does this .to_dict() do? is it helping to convert the plot object attribute into a dict?

Reply:
Yes you can use to_dict(). to check the properties of the plot rather than using encoding, field and etc.